libs/coroutines/include/hpx/coroutines/detail/get_stack_pointer.hpp

```diff
@@ -45,6 +48,8 @@ namespace hpx { namespace threads { namespace coroutines { namespace detail {
     }
 }}}}    // namespace hpx::threads::coroutines::detail
 
+#else
+#define HPX_HAVE_THREADS_GET_STACK_POINTER
```

Comment:
Would you mind inverting the `#if defined(HPX_WINDOWS)` and moving this `#define` up to the top? I think it makes the logic easier to follow.

Reply:
Done.